GNUmakefile

```diff
@@ -228,10 +229,15 @@ test-nomad: dev ## Run Nomad test suites
 	@echo "==> Running Nomad test suites:"
 	@NOMAD_TEST_RKT=1 \
 		go test \
+			-v \
 			-cover \
 			-timeout=900s \
-			-tags="nomad_test $(if $(HAS_LXC),lxc)" \
-			./...
+			-tags="nomad_test $(if $(HAS_LXC),lxc)" ./... >test.log ; echo $$? > exit-code
+	@echo "Exit code: $$(cat exit-code)" >> test.log
+	@grep -A1 -- '--- FAIL:' test.log || true
+	@grep '^FAIL' test.log || true
```

Comment:
Does this test file persist after the task has finished? If so, adding it to the .gitignore would be good.

Reply:
*.log is already ignored https://github.com/hashicorp/nomad/blob/master/.gitignore#L5

Reply:
Would searching for a panic and `# github.com` when there are build errors be useful?

Reply:
Panic is a good one to grep out, I will add that. Test build errors are already caught by the `^FAIL` grep above, and the test output will show something like `FAIL	github.com/hashicorp/nomad/nomad/deploymentwatcher [build failed]
`.